site/content/blogs/release-v125.en.md

```diff
@@ -121,7 +156,133 @@ Mappings:
 ```
 
 ### Integrating With Workloads
-todo @wanxiay
+You can reference values from your environment addons in your workload-level resources. 
+
+#### Reference Environment Addons Values In Workload Addon
+
+##### Step 1: Export values from your environment addons
+In an environment addon template, you should add an `Outputs` section, and define the `Output` that you want your 
+workload resource to reference to. See [this doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html) for CloudFormation `Outputs` syntax.
+
+Taking the example template that we provided - this is the `Outputs` section that we have added in the example.
+```yaml
+Outputs:
+  MyTableARN:
+    Value: !GetAtt MyTable.Arn
+    Export:
+      Name: !Sub ${App}-${Env}-MyTableARN
+  MyTableName:
+    Value: !Ref MyTable
+    Export:
+      Name: !Sub ${App}-${Env}-MyTableName
+```
+
+You can specify any name you like for `Export.Name`. However, the name must be unique within an AWS region; therefore 
+we recommend you to namespace it with `${App}` and `${Env}` to reduce the chances of name collision. 
+With the namespace, for example, say your application's name  is `"my-app"`, 
+and you deployed the addons with environment `test`, then the final export name would be `my-app-test-MyTableName`.
+
+After you've made the code change, run `copilot env deploy` for the change to take effect.
+
+
+##### Step 2: Import the values from your workload addons.
+
+In your workload addons, use [`Fn::ImportValue`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-importvalue.html) function to import the value that you just exported from your environment addons.
+
+Continuing the example that we had. Say now I want my `db-front` service to access `MyTable`, I will create a workload addon 
+attached to `db-front`, with an IAM policy that gives it the access.
+
+```yaml
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+Resources:
+  MyTableAccessPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Description: Grants CRUD access to the Dynamo DB table
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: DDBActions
+            Effect: Allow
+            Action:
+              - dynamodb:* # NOTE: Scope down the permissions in your real application. This is done so that the blog isn't too long!
```

Comment:
```suggestion
              - dynamodb:* # NOTE: Scope down the permissions in your real application. This is done so that this blog post isn't too long!
```